sources/packages/web/src/views/institution/locations/active-applications/LocationActiveApplicationSummary.vue

```diff
@@ -72,15 +75,12 @@ export default {
       type: Number,
       required: true,
     },
-    locationName: {
-      type: String,
-      required: true,
-    },
   },
   setup(props: any) {
     const router = useRouter();
     const { dateString } = useFormatters();
     const applications = ref([] as ActiveApplicationSummaryAPIOutDTO[]);
+    const locationDetails = ref();
```

Comment:
Location details can be typed with the return type from `getInstitutionLocation`.